app/controllers/members_controller.rb

```diff
@@ -45,6 +47,22 @@ def self.add_tab_script(script)
     @@scripts.unshift(script)
   end
 
+  def index
+    @roles = Role.find_all_givable
+    # Check if there is at least one principal that can be added to the project
+    @principals_available = @project.possible_members("", 1)
+
+    @members = @project.member_principals.includes(:roles, :principal, :member_roles)
+                 .order(User::USER_FORMATS_STRUCTURE[Setting.user_format].map{|attr| attr.to_s}.join(", "))
```

Comment:
Line is too long. [107/100]<br>Align the operands of an expression in an assignment spanning multiple lines.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.<br>Pass `&:to_s` as an argument to `map` instead of a block.